charts/consul/templates/crd-exportedservices.yaml

```diff
@@ -1,21 +1,18 @@
 {{- if .Values.connectInject.enabled }}
-# Copyright (c) HashiCorp, Inc.
-# SPDX-License-Identifier: MPL-2.0
-
```

Comment:
Do we want to bring back the license headers? Probably the generation script blew them away.

Reply:
Non-blocking